src/test/url-handling.test.js

```diff
@@ -214,6 +214,17 @@ describe('url upgrading', function() {
     });
   });
 
+  describe('version 3: remove platform only option', function() {
+    it('switches to the stack chart when given a timeline tab', function() {
+      const { getState } = _getStoreWithURL({
+        pathname: '/public/e71ce9584da34298627fb66ac7f2f245ba5edbf5/timeline/',
+        search: '?hidePlatformDetails&v=2',
+        v: false,
```

Comment:
nit: use `v: 2` instead of adding it to the `search` above.
This works the same but I believe it's more scannable and straightforward